llvm/include/llvm/MC/MCTargetOptions.h

```diff
@@ -68,6 +68,8 @@ class MCTargetOptions {
   // ELF.
   bool X86RelaxRelocations = true;
 
+  bool SSE2AVX : 1;
```

Comment:
The size isn't an issue, so I think bit fields are not needed, especially if the bit is going to be used a lot.
If this is not hot and bit fields are intended, this should be placed before.

Reply:
Done